readthedocs/__init__.py

```diff
@@ -0,0 +1,5 @@
+from __future__ import absolute_import
+
+# This will make sure the app is always imported when
+# Django starts so that shared_task will use this app.
+from .settings.celery import app as celery_app  # noqa
```

Comment:
Yes, my plan is to move this to an `AppConfig.ready()` method, but I haven't had a chance to try it with shared tasks yet.